src/components/Root/index.js

```diff
@@ -3,32 +3,37 @@ import 'babel-polyfill'
 
 import { MuiThemeProvider } from '@material-ui/core/styles'
 import { ConnectedRouter } from 'connected-react-router'
-import React, { Suspense } from 'react'
+import React, { Suspense, useState } from 'react'
 import { hot } from 'react-hot-loader/root'
 import { Provider } from 'react-redux'
 
 import Loader from '../Loader'
 import PageFrame from '../layout/PageFrame'
 
-import AppRoutes from '~/routes'
+const AppRoutes = React.lazy(() => import('~/routes'))
```

Comment:
Why lazy load the routes? Every individual route is lazy loaded and I think lazy loading the routes logic is actually worsens the experience since you have to load a logic for routes to load a route

Reply:
fixed